subprojects/docs/src/samples/userguide/dependencies/dependencyConstraints/build.gradle

```diff
@@ -0,0 +1,20 @@
+apply plugin: 'java-library'
+
+repositories {
+    mavenCentral()
+}
+
+//START SNIPPET dependency-constraints
+dependencies {
```

Comment:
What's the difference to the other sample project?

Reply:
They are supposed to demonstrate two different use cases for dependency constraints.

This one includes a constraint for a transitive dependency. The one above is a about a direct dependency declaration. It shows using a dependency constraint to declare the version separately from the actual dependency declaration.

Reply:
:o: I think it would make sense to use a more specific directory name here for the samples that indicates the use case.

Reply:
Done